### ISSUE TITLE
intercom-rails 0.2.36 breaks rake tasks with LoadError for intercom.rake

### ISSUE THREAD

**Issue by mattbrictson** [intercom/intercom-rails#210]
The `intercom.rake` file was removed in 0.2.36, but the `load 'intercom-rails/intercom.rake'` line in the railtie was not removed. That means any Rake task will crash with a LoadError as soon as the intercom-rails railtie is loaded by Rails.

## Version info
  - intercom-rails version: 0.2.36
  - Rails version: 4.2.6

## Expected behavior

`rake test` and other Rake tasks should execute without error.

## Actual behavior

```
$ rake test
rake aborted!
LoadError: cannot load such file -- intercom-rails/intercom.rake
Rakefile:6:in `<top (required)>'
```

## Steps to reproduce
 1. Add `gem "intercom-rails", "0.2.36"` to Gemfile
 2. `bundle install`
 3. `rake -T`

## Logs

```
LoadError: cannot load such file -- intercom-rails/intercom.rake
activesupport-4.2.6/lib/active_support/dependencies.rb:268:in `load'
activesupport-4.2.6/lib/active_support/dependencies.rb:268:in `block in load'
activesupport-4.2.6/lib/active_support/dependencies.rb:240:in `load_dependency'
activesupport-4.2.6/lib/active_support/dependencies.rb:268:in `load'
intercom-rails-0.2.36/lib/intercom-rails/railtie.rb:15:in `block in <class:Railtie>'
```

**Comment by mattbrictson** [comment]
I would submit a PR to fix this, but there appears to be no branch for the 0.2.x series.

**Comment by Skaelv** [comment]
This has been fixed in the 0.3.x versions.
